packages/app-project/src/components/ProjectHeader/ProjectHeaderContainer.spec.js

```diff
@@ -9,11 +9,23 @@ let wrapper
 let componentWrapper
 
 const PROJECT = projects.mocks.resources.projectOne
+const OWNER = 'foo'
+const PROJECT_SLUG = 'bar'
```

Comment:
Perhaps we shouldn't be calling this the slug, because that's the concatenation of the owner and display_name serialized from the API for us on the project resource. This could be confusing. Maybe `PROJECT_DISPLAY_NAME`?